grits.sh

```diff
@@ -26,15 +28,15 @@ fi
 ln -s $(pwd)/geonames.tar /var/log/geonames.tar
 cd /var/log/ && tar -xf geonames.tar &&\ 
 docker exec -t mongodb mongorestore --db geonames /var/log/geonames
+rm geonames.tar /var/log/geonames.tar
 
 #Ensure we have a copy of the grits image
-if [[ ! -f grits.tar.gz && ! -f grits.tar ]]; then
-  aws s3 cp s3://bsve-integration/grits.tar.gz ./grits.tar.gz
-  gzip -d grits.tar.gz
-fi
+aws s3 cp s3://bsve-integration/grits.tar.gz ./grits.tar.gz
+gzip -d grits.tar.gz
 
 #Load the image
 docker load < grits.tar
+rm grits.tar
```

Comment:
This should be done before the gzip -d command. If a grits.tar is left over from before the update the script will wait for user input to confirm it should be overwritten.